src/classes/RD2_ScheduleService_TEST.cls

```diff
@@ -170,22 +170,173 @@ private with sharing class RD2_ScheduleService_TEST {
     private static void shouldNotCreateAScheduleWhenClosed() {
 
         npe03__Recurring_Donation__c rds =
-            TEST_RecurringDonationBuilder.constructEnhancedBuilder()
-                .withMockId()
-                .withContact(UTIL_UnitTestData_TEST.mockId(Contact.SObjectType))
-                .withAmount(400)
-                .withPaymentMethod('Check')
-                .withInstallmentPeriodWeekly()
-                .withInstallmentFrequency(1)
-                .withDateEstablished(Date.newInstance(2019, 9, 15))
-                .withStartDate(Date.newInstance(2019, 11, 1))
-                .withRecurringDonationStatusClosed()
-                .build();
+                TEST_RecurringDonationBuilder.constructEnhancedBuilder()
+                        .withMockId()
+                        .withContact(UTIL_UnitTestData_TEST.mockId(Contact.SObjectType))
+                        .withAmount(400)
+                        .withPaymentMethod('Check')
+                        .withInstallmentPeriodWeekly()
+                        .withInstallmentFrequency(1)
+                        .withDateEstablished(Date.newInstance(2019, 9, 15))
+                        .withStartDate(Date.newInstance(2019, 11, 1))
+                        .withRecurringDonationStatusClosed()
+                        .build();
 
         List<RecurringDonationSchedule__c> schedules = new RD2_ScheduleService().getNewSchedules(rds);
 
         System.assertEquals(schedules.size(), 0, 'Number of RDs should be 0');
```

Comment:
`System.assertEquals(0, schedules.size(), 'Number of schedule records should match');`